test/persistence-hooks.test.js

```diff
@@ -14,5 +14,10 @@ if (process.env.MONGODB_VERSION &&
 for(var i in config) {
   customConfig[i] = config[i];
 }
+var DB_VERSION = process.env.MONGODB_VERSION;
+var DB_HAS_2_6_FEATURES = (!DB_VERSION ||
```

Comment:
Could you please add a `console.log` to let the users know about this fact?

For example:

```js
if (!DB_VERSION) {
  console.log('The ENV variable MONGODB_VERSION is not set. Assuming MongoDB version 2.6 or newer.');
}
```